defs/panda.def

```diff
@@ -168,6 +134,55 @@ void setXYZE(double px, double py, double pz, double e) override
   mass_ = std::sqrt(e * e - p * p);
 }
 
+% Note that SecondaryVertex stuff is copy-pasted from Vertex and RecoVertex
+% Multiple inheritance isn't working due to inheritance from Element
+[SecondaryVertex>ParticleM]
+x/F
+y/F
+z/F
+TVector3 position() const { return TVector3(x, y, z); }
+ntrk/s
+ndof/F
```

Comment:
Is ndof a float?

Reply:
It is a float under RecoVertex. Like I said in the comments, those are just copy pasted from there. Also, VertexCompositeCandidate.vertexNdof() returns a double. It looks like this is because a track can contribute fractional weights: http://cmsdoxygen.web.cern.ch/cmsdoxygen/CMSSW_8_0_28/doc/html/dc/d3b/classreco_1_1VertexCompositeCandidate.html#a96955a6a7b1ed93815064759b3d521bd

Reply:
Ah OK, I had forgot about that. That makes sense.